test/libsolidity/syntaxTests/natspec/docstring_udvt_definition.sol

```diff
@@ -0,0 +1,8 @@
+contract C {
+    /// @title example of title
+    /// @author example of author
```

Comment:
Not sure `@title` and `@author` make sense here. They were meant for contracts. But I think we already allowed them for structs and enums so I guess for consistency it's better to allow them here too.

The table listing what is usable where should be updated though: https://docs.soliditylang.org/en/latest/natspec-format.html#tags

Also, is `@custom` allowed? The docs say it's allowed everywhere, but if it is, it would be better to include it in the test to show that.